pom.xml

```diff
@@ -62,6 +62,7 @@
         <module>notification-service</module>
         <module>pattern-detection-library</module>
         <module>coverage</module>
+        <module>tasks</module>
```

Comment:
This module is going to be used for all common tasks
I think it should be renamed to more informative name
I suggest: `workflow-common-tasks`

Reply:
We know it is common. We know it is for workflow. Why add all this text? I can somehow understand the addition of 'workflow-'. Some how ...

Reply:
We know its `common`, I'm not sure how clear it would be for new users, I think its better to make it explicit so it would more clear.
Anyway, I don't have any problem with changing to `workflow-tasks`

Reply:
@nirarg fixed